cves/CVE-2017-11665.yml

```diff
@@ -64,7 +64,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A function allowed remote Real Time Messaging Protocol servers to cause a 
```

Comment:
Need a lot more here. Your audience won't know what RTMP is. How does the data turn into triggering a segfault, under what situations would this happen? How might an attacker use this?